<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transfer_card</class>
 <widget class="QWidget" name="Transfer_card">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="step_1_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_holder_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
selection-color: rgb(25, 83, 117);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="maxLength">
    <number>20</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvv_edit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 19pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
selection-color: rgb(25, 83, 117);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="step_2_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="step_3_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_1_img">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>461</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>—Åredit__card.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="header_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Sender's card details</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_3_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 3 of 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_2_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 2 of 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_1_label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Step 1 of 3</string>
   </property>
  </widget>
  <widget class="QLabel" name="card_holder_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>CARD HOLDER</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>255</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>VALID</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>264</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>THRU</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_numb_edit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string>xxxx   xxxx   xxxx   xxxx</string>
   </property>
   <property name="text">
    <string>         </string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string> XXXX   XXXX   XXXX   XXXX</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
selection-color: rgb(25, 83, 117);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2032</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>MM/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="cvv_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>160</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
  </widget>
  <widget class="QLabel" name="step_2_img">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>59</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>credit_card(receiver).png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="card_numb_edit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string>xxxx   xxxx   xxxx   xxxx</string>
   </property>
   <property name="text">
    <string>         </string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="dragEnabled">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string> XXXX   XXXX   XXXX   XXXX</string>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);
selection-color: rgb(3, 224, 249);
selection-background-color: rgb(3, 224, 249);
selection-color: rgb(25, 83, 117);
gridline-color: rgb(25, 83, 117);
border-color: rgb(25, 83, 117);
background-color: rgb(25, 83, 117);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhPreferNumbers</set>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="maximumDateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>2032</year>
     <month>10</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </datetime>
   </property>
   <property name="minimumDate">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>MM/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>false</bool>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="date_label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>VALID</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_label_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 7pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>THRU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Segoe UI&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Transfer from card to card</string>
   </property>
  </widget>
  <zorder>step_1_img</zorder>
  <zorder>step_3_button</zorder>
  <zorder>card_holder_edit</zorder>
  <zorder>cvv_edit</zorder>
  <zorder>step_2_button</zorder>
  <zorder>header_label</zorder>
  <zorder>step_3_label</zorder>
  <zorder>step_2_label</zorder>
  <zorder>step_1_label</zorder>
  <zorder>step_1_button</zorder>
  <zorder>card_holder_label</zorder>
  <zorder>date_label_2</zorder>
  <zorder>date_label_3</zorder>
  <zorder>card_numb_edit</zorder>
  <zorder>date_edit</zorder>
  <zorder>cvv_label</zorder>
  <zorder>step_2_img</zorder>
  <zorder>card_numb_edit_2</zorder>
  <zorder>date_edit_2</zorder>
  <zorder>date_label_4</zorder>
  <zorder>date_label_5</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
